connectors/base/base.go

```diff
@@ -188,3 +190,8 @@ func (c *Connector) Shutdown() error {
 	}
 	return c.Next.Shutdown()
 }
+
+// Name returns the name of the connector
+func Name() string {
```

Comment:
I don't fully understand this comment. Connectors "register" themselves when the init() function on the connector is called. Then, to create the connector, you just call dosa.GetConnector with the name. At that point, the name is already registered with the connector registry.